yajsapi/payment/service.ts

```diff
@@ -83,6 +83,7 @@ export class PaymentService {
     this.payments?.unsubscribe().catch((error) => this.logger?.warn(error));
     this.payments?.removeEventListener(PaymentEventType, this.subscribePayments.bind(this));
     for (const allocation of this.allocations) await allocation.release().catch((error) => this.logger?.warn(error));
+    this.options.httpAgent.destroy?.();
```

Comment:
Why do you need the `?` here? This method is always present if you built the HTTP agent correctly in the constructor. I checked this locally and the code compiles `this.options.httpAgent.destroy();` without any issues.

Reply:
There is a polyfill in the browser and unfortunately it does not have this method:

https://github.com/FredKSchott/rollup-plugin-polyfill-node/blob/eb507a21ad82236255988548dea6120d7a277de2/polyfills/http.js#L71

We made httpAgent mainly because in nodejs there were random errors in connections to yagna - socket hangup and httpAgent resolve this problem. It wasn't in the browser.